app.js

```diff
@@ -35,28 +34,11 @@ app.use(async function(ctx, next) {
 const Router = require('koa-router');
 const router = new Router();
 
-const creatorKeyJson = (() => {
-    try {
-        return JSON.parse(process.env.ACCOUNT_CREATOR_KEY);
-    } catch (e) {
-        console.warn(`Account creation not available.\nError parsing ACCOUNT_CREATOR_KEY='${process.env.ACCOUNT_CREATOR_KEY}':`, e);
-        return null;
-    }
-})();
-const keyStore = {
-    async getKey() {
-        return nearAPI.KeyPair.fromString(creatorKeyJson.secret_key || creatorKeyJson.private_key);
-    }
-};
+const {
+    creatorKeyJson,
+    nearPromise,
```

Comment:
`nearPromise` also used only in one middleware, makes sense to export middleware `withNear` instead

Reply:
Answered in https://github.com/near/near-contract-helper/pull/165/files/d70186d6f37625eedf238e048c2a2042e143745f#r445239440